modules/olmv1-clusteroperator.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cluster-operators-ref-olmv1_{context}"]
+= {olmv1-first} Operator
+
+[discrete]
+== Purpose
```

Comment:
Need a unique ID after this discrete heading, I think https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

Reply:
The modules used in the "Cluster Operators reference" assembly (mostly) all follow a similar layout and set of subsections, which all have a `[discrete]` heading with no ID, so I figured I'd match that. FWIW, it doesn't break the build, but it is unclickable and untargetable. 

Random other example on [docs.openshift](https://docs.openshift.com/container-platform/4.17/operators/operator-reference.html#cloud-credential-operator_cluster-operators-ref) vs [docs.redhat](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/operators/cluster-operators-ref#cluster-bare-metal-operator_cluster-operators-ref).